### ISSUE TITLE
paperclip gem fails

### ISSUE THREAD

**Issue by brianlball** [NREL/OpenStudio-server#619]
paperclip 4.3.7 require mimemagic 0.3.0 which got pulled from the interwebs

https://github.com/rails/rails/issues/41750

**Comment by juliobetta** [comment]
hey @brianlball @DavidGoldwasser any ideas on how to fix that? I tried replacing mimemagic version directly in Gemfile.lock from my Dockerfile... off course it didn't work 😅 

```
RUN sed -i 's/mimemagic (0.3.0)/mimemagic (0.3.10)/g' /opt/openstudio/server/Gemfile.lock
RUN sed -i 's/mimemagic (= 0.3.0)/mimemagic (= 0.3.10)/g' /opt/openstudio/server/Gemfile.lock
```

**Comment by brianlball** [comment]
Tim made a fix in this branch for 3.1.0 https://github.com/NREL/OpenStudio-server/tree/3.1.X-LTS
